tests/unit/amo/sagas/test_home.js

```diff
@@ -142,7 +161,7 @@ describe(__filename, () => {
 
       mockSearchApi
         .expects('search')
-        .once()
+        .twice()
```

Comment:
Is this just because we call it twice for different results? Should we test if only one response is bad, as well?

Reply:
Well, either we put `twice` or nothing actually, otherwise sinon will complain.

Reply:
No actually it is `twice` or ... it fails because of `sinon` 😓 

I changed this test to use the collection API, which is used only once so that it keeps the same behavior as before.